kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -0,0 +1,94 @@
+server {
+    listen 80;
+
+    gzip                     on;
+    gzip_min_length          1000;
+    gzip_proxied             expired no-cache no-store private auth;
+    gzip_types               text/plain text/css text/javascript application/xml;
+    client_max_body_size     {{ max_upload_size }};
+
+
+    location = /upload/ {
+        access_by_lua_block {
+            -- We're using nginx-big-upload module which supports only "application/x-www-form-urlencoded".
+            -- Return an error if the content type is different.
+            if ngx.req.get_headers()["Content-Type"] ~= "application/x-www-form-urlencoded" then
+                ngx.status = 415
+                ngx.say('Unsupported content type. The only supported type is application/x-www-form-urlencoded.')
+                ngx.exit(415)
+            end
+
+            -- Make sure that Concent-Upload-Path header with path and filename were included with the request.
+            if ngx.req.get_headers()["Concent-Upload-Path"] == nil then
+                ngx.status = 415
+                ngx.say('Concent-Upload-Path header is missing.')
+                ngx.exit(415)
+            end
+
+            -- Make sure that the request actually has a body.
+            if ngx.req.get_headers()["Content-Length"] == "0" then
+                ngx.status = 415
+                ngx.say('The file to be uploaded must not be empty.')
+                ngx.exit(415)
+            end
+        }
+
+        # Strip the X-Session-Id header from the request if client has provided it.
+        # nginx-big-upload uses his header to set the name of the temporary file.
+        # If it's empty, a random name will be generated and that's exactly what we want.
+        proxy_set_header X-Session-Id "";
+        proxy_pass       http://nginx-storage/upload-internal/;
+    }
+
+    location /upload-internal/ {
+        lua_code_cache   on;
+        set $storage     backend_file;
+        set $bu_checksum on;
+        set $bu_sha1     off;
+
+        set $file_storage_path /srv/storage;
+        set $backend_url       /result;
+        set $package_path      '/opt/nginx-big-upload/lib/?.lua';
+        content_by_lua_file    /opt/nginx-big-upload/lib/big-upload.lua;
+
+    }
+
+    location /result {
```

Comment:
The trailing `/` is missing here. Should be `/result/` for consistency with other endpoints.

Reply:
Done.